packages/terra-arrange/src/terra-dev-site/doc/example/ArrangeAlignAllContainers.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+/* eslint-disable import/no-extraneous-dependencies */
+import Arrange from 'terra-arrange/lib/Arrange';
+import { alignExampleDivBlue, textWithBlueBorder } from '../common/examplesetup';
+import styles from './ArrangeExamples.module.scss';
+
+const cx = classNames.bind(styles);
+
+const ArrangeAlignAllContainers = () => (
+  <div>
+    <h4>Align - Default</h4>
```

Comment:
Is this the correct heading for accessibility of this page?

Reply:
Not sure. Heading might not even be the appropriate way to set it up, so I’ll look into it.

Reply:
An `h3` element here would better match the document outline.

Reply:
Updated: [68c3a20](https://github.com/cerner/terra-core/pull/2353/commits/68c3a206ca8b941ea1e9f9e7a3e6fad41525c455).